<launch>
    <arg name="gui" default="true"/>
    <arg name="args" default=""/>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find mavdrone_rl)/config/mavdrone_test.yaml" />
    <!-- Launch the training system -->
    <!-- <include file="$(find mavdrone_rl)/launch/env.launch" /> -->
    <node pkg="mavdrone_rl" name="mavdrone_test_env" type="test_env.py" output="screen"/>
  </group> -->
</launch>
